client/state/current-user/gravatar-status/actions.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import request from 'superagent';
+
+import {
+	GRAVATAR_UPLOAD_RECEIVE,
+	GRAVATAR_UPLOAD_REQUEST,
+	GRAVATAR_UPLOAD_REQUEST_SUCCESS,
+	GRAVATAR_UPLOAD_REQUEST_FAILURE
+} from 'state/action-types';
+
+export function uploadGravatar( file, bearerToken, email ) {
+	return dispatch => {
```

Comment:
I'm not sure we're being so strict on that. The entire section is being removed in #9037. I would say always aim for readability. I tend to only omit parens if I'm writing a one liner, but I don't really see the harm in this case.